src/Kriti/Eval.hs

```diff
@@ -47,13 +52,22 @@ instance SerializeError EvalError where
   serialize (InvalidPath _ term path) = SerializedError {_code = InvalidPathCode, _message = "\"" <> renderVect path <> "\"", _span = locate term}
   serialize (TypeError _ term msg) = SerializedError {_code = TypeErrorCode, _message = msg, _span = locate term}
   serialize (RangeError _ term) = SerializedError {_code = RangeErrorCode, _message = "Can only range over an array", _span = locate term}
+  serialize (FuncNotFound _ term msg) = SerializedError {_code = TypeErrorCode, _message = msg, _span = locate term}
+  serialize (CustomError msg) = SerializedError {_code = CustomErrorCode, _message = msg, _span = dummySpan}
+
+dummySpan :: Span
+dummySpan = Span dummyPos dummyPos
+  where
+    dummyPos = AlexSourcePos 0 0
 
 type Ctxt = (B.ByteString, Compat.Object J.Value)
 
 getSourcePos :: EvalError -> Span
 getSourcePos (InvalidPath _ pos _) = locate pos
 getSourcePos (TypeError _ term _) = locate term
 getSourcePos (RangeError _ pos) = locate pos
+getSourcePos (FuncNotFound _ term _) = locate term
+getSourcePos (CustomError _) = dummySpan
```

Comment:
We should include an actual span in `CustomError` so that we don't need `dummySpan`.

Reply:
Done